src/lib/components/grades/gradeCard.svelte

```diff
@@ -0,0 +1,66 @@
+<script lang="ts">
+	import * as allIonicIcons from 'ionicons/icons';
+	import type { course } from './courseType';
+	/**
```

Comment:
JSDOC Typedef isn't needed here, as we're importing the type from CourseType.ts lines 4-6

Reply:
fixed